src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -59,7 +59,7 @@ private class Searcher
                 if (_searchCurrentDocument)
                 {
                     var documentService = _solution.Workspace.Services.GetService<IDocumentTrackingService>();
-                    var activeId = documentService.GetActiveDocument();
+                    var activeId = documentService.TryGetActiveDocument();
                     _currentDocument = activeId != null ? _solution.GetDocument(activeId) : null;
```

Comment:
up to you, but the null checks are unnecessary here.  Workspace is defined to return a null doc on a null docId.

Reply:
Heh, proves that the rename @sharwell wanted makes sense. Honestly, going to leave it as is, since I think GetDocument accepting a null ID isn't good to start with...